ros2_ws/setup_submodules.bash

```diff
@@ -0,0 +1,10 @@
+git submodule init
+git submodule update
+
+if [ "$ROS_DISTRO" == "iron" ]
+then
+	cd ./src/Universal_Robots_ROS2_Driver/
+	git checkout 129687763140508b4b5cb497b197ce1057defe88
```

Comment:
Should we also check if the checkout was successful? I can imagine the following situation (I had it):
```bash
jhanca@robotec-o3de:~/devroot/projects/ROSCon2023Demo/ros2_ws$ ./setup_submodules.bash
fatal: reference is not a tree: 129687763140508b4b5cb497b197ce1057defe88
Submodules updated for ROS2 iron.
```

```suggestion
	git checkout 129687763140508b4b5cb497b197ce1057defe88
	if [ $? -ne 0 ]
	then
            echo "Failed to checkout ROS 2 iron branch, submodule updated for ROS 2 humble."
            exit 1
        fi
```

Reply:
Good point, added!